File: server/src/main/java/com/smartexpense/api/application/dto/request/UpdateProfileRequestDto.java
package com.smartexpense.api.application.dto.request;

import lombok.Data;
import java.math.BigDecimal;

@Data
public class UpdateProfileRequestDto {
    private String displayName;
    private String avatarUrl;
    private BigDecimal monthlyBudget;
    private String themeColor;
}

File: server/src/main/kotlin/com/smartexpense/api/application/dto/request/UpdateProfileRequestDto.kt
package com.smartexpense.api.application.dto.request

import java.math.BigDecimal

data class UpdateProfileRequestDto(
    val displayName: String? = null,
    val avatarUrl: String? = null,
    val monthlyBudget: BigDecimal? = null,
    val themeColor: String? = null
)
